.github/workflows/UpdateCodeowners.yml

```diff
@@ -8,6 +8,7 @@ on:
   schedule:
     # https://crontab.guru/#5_8_*_*_1
     - cron: "5 8 * * 1"
+  workflow_dispatch:
```

Comment:
is this empty for now, per purpose?

Reply:
Yep, it can't just be a string, this is a yml empty object I believe